### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by abenaTenkorang**
In this project;

-Use media queries to make sure that your website displays correctly on desktop screens.
-You have time for this project scheduled in to 2 days. Please ask for a code review when you are ready.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,227 +80,223 @@ <h2 class="header-title ">I'm Abigail T Glad to see you!
...
+          </ul>   
+      </div>
+     
   <div class="btn-container">
     <a href="#" class="ctb-btn">Get my resume</a>
   </div>
+</div>
 
   <!-- Languages -->
+  <div class="lang">
```
Comment (bienvenuushindi) [inline index.html:294]: :red_circle: Please make sure you reduce the space between the `about` and `lang` blocks to match the Figma Design

Your Design
your design

Figma Design
enoughspace

File: index.html [hunk 2]
```diff
@@ -65,227 +80,223 @@ <h2 class="header-title ">I'm Abigail T Glad to see you!
...
+
+    </div>
   </div>
+</article>
+ 
 
+</section>
+
+<!-- top section -->
+<section class="top-section">
```
Comment (bienvenuushindi) [inline index.html:244]: :red_circle:  Please make sure you add some space at the top of this section (`<section class="top-section">`) to match the Figma Design

Your Design
noroom

Figma Design
room

File: index.html [hunk 3]
```diff
@@ -65,227 +80,223 @@ <h2 class="header-title ">I'm Abigail T Glad to see you!
...
+        />
+    </div>
+    <div class="card-content">
+      <h2 class="card-title">Multi-Post Stories</h2>
+      <div class="card-info">
+        <p class="card-info-item-canopy">FACEBOOK</p>
+        <img class="circle-img" src="./images/circle.png" alt="Circle">
+        <p class="card-info-item">Full Stack Dev</p>
+        <img class="circle-img" src="./images/circle.png" alt="Circle">
+        <p class="card-info-item">2015</p>
```
Comment (bienvenuushindi) [inline index.html:144]: :red_circle: Please make sure you increase the space between the project's picture and the project's description to match the Figma Design
As you can see below :arrow_down: Your design lacks enough space between the project picture and  description on the project cards where the project description is aligned to the right. You can use `nth-child(even)` to target these elements whose index is even  (the index of the first child is 1).

For example 
```
.card-content:nth-child(even) .card-title {
   margin-right: 20px;
}

```
|Your Design|Figma  Design|
| :---: | :---:  |
|y|f|

File: index.html [hunk 4]
```diff
@@ -65,227 +80,223 @@ <h2 class="header-title ">I'm Abigail T Glad to see you!
...
+
+        <ul class="card-list">
+          <li class="card-list-tags-html">html</li>
+          <li class="card-list-tags-css">css</li>
+          <li class="card-list-tags-js">Javascript</li>
+        </ul>
+
+        <a class="card-link" href="#">See Project</a>
+
+      </div>
```
Comment (bienvenuushindi) [inline index.html:120]: :red_circle: Please make sure you increase the space between the project's picture and the project's description to match the Figma Design
As you can see below :arrow_down: Your design lacks enough space between the project picture and  description on the project cards where the project description is aligned to the right. You can use[ `nth-child(even)` ](https://www.w3schools.com/csSref/sel_nth-child.php)to target these elements whose index is even  (the index of the first child is 1).



|Your Design|Figma  Design|
| :---: | :---:  |
|y|f|

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @abenaTenkorang ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linter checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (abenaTenkorang) [reply index.html:294]: Issue resolved. Thank you

File: index.html (hunk 2 above)
Comment (abenaTenkorang) [reply index.html:244]: conversation resolved. Thank you for drawing my mind to that

File: index.html (hunk 3 above)
Comment (abenaTenkorang) [reply index.html:144]: Conversation resolved. Thank you

**Review by pinkmoon25 — APPROVED** [review]
Hi @abenaTenkorang , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
You documented your work professionally. :heavy_check_mark:
GitHub flow is used correctly. :heavy_check_mark:
All linter checks are ok. :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**